kratos/geometries/geometry.h

```diff
@@ -1724,6 +1724,28 @@ class Geometry
         return rResult;
     }
 
+    /**
+    * @brief This method maps from dimension space to working space and computes the
+    *        number of derivatives at the dimension parameter.
+    * @param LocalCoordinates The local coordinates in dimension space
+    * @param Derivative Number of computed derivatives
+    * @return std::vector<array_1d<double, 3>> with the coordinates in working space
+    * @see PointLocalCoordinates
+    */
+    virtual void GlobalDerivatives(
+        std::vector<CoordinatesArrayType>& rGlobalCoordinates,
+        const CoordinatesArrayType& rCoordinates,
+        const SizeType DerivativeOrder) const
+    {
+        KRATOS_ERROR << "Calling GlobalDerivatives within geometry.h."
```

Comment:
Would it makes sense to call ˋGeometry::GlobalCoordinatesˋ if ˋDerivativeOrder == 0ˋ?

Reply:
I think you're right. I put also DerivativeOrder == 1 as this information can be easily computed.